python/ray/serve/api.py

```diff
@@ -565,9 +565,15 @@ def run(
             will loop and log status until Ctrl-C'd.
         name: Application name. If not provided, this will be the only
             application running on the cluster (it will delete all others).
-        route_prefix: Route prefix for HTTP requests. If not provided, it will use
-            route_prefix of the ingress deployment. If specified neither as an argument
-            nor in the ingress deployment, the route prefix will default to '/'.
+        route_prefix: Route prefix for HTTP requests. If not provided, it will use the
+            `route_prefix` of the application's ingress deployment.
+            If specified neither as an argument nor in the ingress deployment,
+            the route prefix will default to '/'.
+            If `None`, the application will not be exposed over HTTP
+            (this may be useful if you only want the application to be usable
+            via other methods like
+            [gRPC](serve-set-up-grpc-service) or
+            `serve.get_app_handle()`).
```

Comment:
actually route_prefix on the deployment is deprecated so let's clean this up more:
```suggestion
        route_prefix: Route prefix for HTTP requests. Defaults to '/'.
            If `None` is passed, the application will not be exposed over HTTP
            (this may be useful if you only want the application to be exposed via
            gRPC or a `DeploymentHandle`.
```
suggest to drop the link here because it will render poorly in IDEs/shells